### ISSUE TITLE
IBInspectables

### ISSUE THREAD

**Issue by Mazyod** [muZZkat/NMRangeSlider#42]
Hi,

Thanks for sharing this great component. I would like to suggest adding some `IBInspectable` flags to some properties. Hopefully, it won't take much time (unless the values aren't meant to change after initialization), but will go a long way in customizing the control from IB.

Thanks,
Maz

**Comment by muZZkat** [comment]
Good idea. I'm a bit swamped at the moment but I hope to have some time to spend on this project before the end of the month.
